entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/bash
+source /opt/ros/noetic/setup.bash
+
+# Build & Source catkin_ws
+cd barracuda-thruster-output-server/catkin_ws
+catkin_make
+source devel/setup.bash
+
+# Start the launch file (add --wait after other node starts roscore)
+roslaunch barracuda_thruster_output_server barracuda_thruster_output_server.launch
```

Comment:
should just include --wait tag. the tag will cause it to wait for the roscore to start; this node should not start a roscore

Reply:
would recommend you comment this out for this PR and uncomment the debugging code, since we don't have a catkin_ws and we dont have the ros package created yet.